ansible/advanced.md

```diff
@@ -84,6 +94,20 @@ meanings for different followers.
 * ER-301 can be sent gate and CV commands, with the 4 tracks
   corresponding to the first 4 `SC` channels.
 
+* Disting Ex, left to right: allocate notes/gates polyphonically, map each
+  track to one output, convert CV to MIDI via the MIDI breakout on the
+  Disting EX, all channels sent via channel 1, all channels send
+  via channels 1-4. In MIDI modes data is passed through the Disting,
+  it cannot be used in single algorithm mode, but will function
+  in dual algorithm mode.
+
+* W/ Syn, left to right: allocate notes polyphonically, map each track
+  to one output as a synth voice.
+
+* Crow, sends output number, pitch and duration.
```

Comment:
@csboling , this is all correct to the most recent changes, yeah?

Reply:
@csboling I'd love to know the intended use case and outcome for Crow in this situation, and now it relates to the the 4 tracks in the ii configuration page.

Reply:
The idea is that rather than any predefined behavior on Crow, Ansible causes Crow to call its `ii.self.call` function with the track number, pitch, and "velocity" values, so that depending on what Lua code Crow is running you can have Ansible's note events cause arbitrary behavior or note processing. I admit I have not used this functionality a ton so there may be a more useful thing to do but this seemed like a good tradeoff of flexibility for implementation complexity.

Reply:
ah gotcha -- dang, that sounds super rad. i'll give it a spin and see if i can't supplement docs with some code examples on the crow side.